<robot name="architect_model">
  <link name="marker2">
    <visual>
      <origin rpy="0 0 0" xyz="0.0 0 0.0"/>
      <geometry>
        <mesh filename="file:///home/johnny/datasets/hilti/cla.stl"/>
      </geometry>
    </visual>
  </link>
</robot>